data/Language/Jinja2.yaml

```diff
@@ -0,0 +1,44 @@
+identifier: Jinja2
+full_name: Jinja2
+wikidata: Q4041851
+extensions:
+  - jj2
+  - j2
+  - jinja
+  - jinja2
+delimiters:
+  # comment delimiter
+  - multiline_curly_braces_hash
+  # block delimiter
+  - curly_braces_curly_braces
+  - curly_braces_percent
+keywords:
+  - as
+  - block
+  - call
+  - else
+  - endblock
+  - endcall
+  - endfilter
+  - endfor
+  - endif
+  - endmacro
+  - endraw
+  - endset
+  - endwith
+  - extends
+  - 'False'
```

Comment:
improve the model for datatype (which was created by GCI student) so that a datatype can have a list of literals, and for this it would be `True` and `False` , and for some other languages it would be `true` & `false` .

For Jinja2 & Python, create a datatype: bool_title_case , and then remove True and False from the generic keywords listed here.

Reply:
https://github.com/coala/coAST/pull/26 did this for you ...

Reply:
Since the `bool_title_case` datatype has been added. Would it be a right move to remove the `bool_python` from the `datatypes` since `bool_title_case` is generic and can be used by both the languages as specified by you.

Is it wise to create a issue for it?